lib/vitals/reporters/riemann_reporter.rb

```diff
@@ -0,0 +1,37 @@
+require 'riemann/client'
+
+module Vitals::Reporters
+  class RiemannReporter < BaseReporter
+    attr_accessor :format
+    attr_reader :riemann
+
+    def initialize(host: 'localhost', port: 5555, timeout: 5, format:nil)
+      @riemann = Riemann::Client.new(host: host, port: port, timeout: timeout)
```

Comment:
The Riemann client does not open the connection(which is slowish) until the first metric is sent. Do we want to warm the connection by sending a 'connected' even to Riemann?